core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sink/storage/MongoDBStorage.scala

```diff
@@ -22,7 +22,6 @@ class MongoDBStorage(id: String, schema: Schema) extends SinkStorageReader {
   val client: MongoClient = MongoClients.create(url)
   val database: MongoDatabase = client.getDatabase(databaseName)
   val commitBatchSize: Int = AmberUtils.amberConfig.getInt("storage.mongodb.commit-batch-size")
-  database.getCollection(id).drop()
```

Comment:
why is this removed?

Reply:
this change is from the retrieval branch not mine